RELEASE_NOTES.md

```diff
@@ -1,13 +1,17 @@
 Release Notes
 =============
 
-v1.0.0-alpha.x
+v1.0.0-alpha.9
 --------------
 
 ### Breaking Changes
 
 - Minimum OpenAssetIO version increased to `v1.0.0-alpha.13` due to API
-  changes.
+  changes. In particular, `getRelatedReferences` has changed to
+  `getWithRelationship` and `getWithRelationships`.
```

Comment:
[Fix] I don't personally think it's appropriate to make specific mention of openassetio methods in the BAL release notes. This isn't a consideration that BAL consumers need to care about, in theory. What do you think?

Reply:
My thought was slightly selfish, in that I've found myself looking at release notes trying to remember whether the `getWithRelationship` rename was already released or not, but it wasn't mentioned at all in the release notes.

Reply:
Dropped the commit that updated the release notes.

Reply:
Thanks for that, sorry for being a pain.